.github/workflows/build_test.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build Test
+on: [push]
+
+jobs:
+  build:
+    name: Validate Changes
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.8]
```

Comment:
Do we still need to bother with Python 2.7?  I know we historically had it there, but I'm not sure there is much value in it anymore.  Thoughts?

Reply:
I'm happy to remove it. I don't think we really need it long term. But I tried to change as little as possible to make sure we're still getting the same results.

Reply:
All right, that's fine.  We can leave it for now, and remove it in a separate PR.